www/source/partials/tutorials/_web_guide_define_dependencies.slim

```diff
@@ -33,7 +33,7 @@ section
 
  p Clicking on one of the entries will show you all of the versions of that package that have been uploaded to Builder. By convention, version numbers align with the version of the binary, library, or framework that the package has bundled up. For example, the <code>core/node</code> version 6.10.3 package has v6.10.3 of the Node.js binary.
 
- p The Builder UI also provides channel information about the package, as denoted by the <code>stable</code> and <code>unstable</code> labels. Channels function like continuous delivery stages in a pipeline (development, QA, production, etc.). When packages are initially uploaded to Builder, they are placed in the <code>unstable</code> channel, and by default, only <code>stable</code> packages are downloaded and installed in a given container, virtual machine, etc unless otherwise specified through the <code>hab start</code> and <code>hab svc load</code> commands. By convention, the latest version number with the latest build date will be the stable version of the package.
+ p The Builder UI also provides channel information about the package, as denoted by the <code>stable</code> and <code>unstable</code> labels. Channels function like continuous delivery stages in a pipeline (development, QA, production, etc.). When packages are initially uploaded to Builder, they are placed in the <code>unstable</code> channel, and by default, only <code>stable</code> packages are downloaded and installed in a given container, virtual machine, etc unless otherwise specified through the <code>hab sup run</code> and <code>hab svc load</code> commands. By convention, the latest version number with the latest build date will be the stable version of the package.
```

Comment:
This has already been fixed, but with different prose. See https://www.habitat.sh/demo/windows/steps/3/